cli/dcoscli/marathon/main.py

```diff
@@ -25,6 +25,7 @@
     dcos marathon group show [--group-version=<group-version>] <group-id>
     dcos marathon group remove [--force] <group-id>
     dcos marathon group update [--force] <group-id> [<properties>...]
+    dcos marathon group scale [--force] <group-id> <scale-factor>
```

Comment:
Can we sort this line? It looks like it should go before `dcos marathon group show`